package.json

```diff
@@ -5,13 +5,15 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint *.js src/*.js",
-
     "run": "node src/index.js"
   },
   "repository": "https://github.com/ProvidenceGeeks/website-lambda-meetup.git",
   "author": "mdiwale@kenzan.com",
   "license": "MIT",
   "devDependencies": {
     "eslint": "^4.8.0"
+  },
+  "dependencies": {
+    "request": "^2.83.0"
```

Comment:
I think by default NodeJS has an `http` module.  What is the advantage of using `request` instead, and did we confirm it ships with NodeJS and is supported by Lambda?

Reply:
@thescientist13 
As per what I read on internet, `request` is supported in Node.JS as well as in Lambda. The reason I used `request` is that I felt it more aligned with the promises and required very few steps than `http` to solve our purpose.

Reply:
great, thanks for confirming @mdiwale !  Let's try and catch up on next steps with this tomorrow, re: uploading the _.json_ file to S3